docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx

```diff
@@ -0,0 +1,829 @@
+### Building a Web-Based User Experience
+
+The Anchor Platform does not offer a whitelabel UI that your business can utilize, and instead expects the business to build their own UI and backend system. We won't build an entire on & off-ramp user experience in this guide, but will cover the ways in which your existing product should be updated to be compatible with the Anchor Platform.
+
+#### Authentication
+
+If your business has an existing on & off-ramp product, you likely have an existing system for user authentication. However, because the Anchor Platform authenticates the user prior to providing the business' URL, requiring the user to go through another form of authentication is actually unnecessary. In this way, the Anchor Platform can be thought of as providing an alternative form of authentication.
+
+The business is free to continue requiring users to authenticate using their existing system, but the ideal user experience would skip this step and create an authenticated session for the user if they have already authenticated using their Stellar account.
+
+The Anchor Platform adds a JWT `token` query parameter to the business' URL given to the wallet application. This token is signed by the previously-configured `SECRET_SEP24_INTERACTIVE_URL_JWT_SECRET` value, and includes the information you need to identify the user. The process should look something like this:
+
+1. Pass the `token` added to the URL of your backend system
+2. Verify the signature on the `token` and check its expiration
+3. Fetch the transaction data using the token's `jti` from the Anchor Platform
+4. Create an authenticated session for the user identified in the transaction
+
+The decoded contents of the `token` will look something like this:
+
+<CodeExample>
+
+```json
+{
+  "jti": "e26cf292-814f-4918-9b40-b4f76a300f98",
+  "sub": "GB244654NC6YPEFU3AY7L25COGES445P3Q63W6Q76JHR3UBJMLT2XBOB:1234567",
+  "exp": 1516239022,
+  "data": {
+    "first_name": "John",
+    "last_name": "Doe",
+    "email": "johndoe@example.com"
+  }
+}
+```
+
+</CodeExample>
+
+Note that the `sub` value identifies the user using a Stellar account and integer. This is what the value will be when custodial applications that use an omnibus account authenticate with your service. When non-custodial wallets authenticate, the token may look slightly different.
+
+<CodeExample>
+
+```json
+{
+  "jti": "e26cf292-814f-4918-9b40-b4f76a300f98",
+  "sub": "GB244654NC6YPEFU3AY7L25COGES445P3Q63W6Q76JHR3UBJMLT2XBOB",
+  "exp": 1516239022,
+  "data": {
+    "client_domain": "api.vibrantapp.com",
+    "first_name": "John",
+    "last_name": "Doe",
+    "email": "johndoe@example.com"
+  }
+}
+```
+
+</CodeExample>
+
+The `sub` value here only contains a public key to identify the user, and the `data.client_domain` field identifies the wallet application used to authenticate.
+
+In both cases, all information in the `data` object is optional, and will only be present if the wallet provides that information.
```

Comment:
Should it be mentioned that data is configurable, and it's possible to include transaction fields too?

Reply:
We can provide more detail when we refactor these docs, there are lots of places where I could dive into more detail